build-native.cmd

```diff
@@ -0,0 +1,3 @@
+@call %dp0buildvars-setup.cmd %*
+@call %~dp0run.cmd build-managed %cleanBuild% -Platform=%__BuildArch% -Configuration=%__BuildType% 
```

Comment:
Do we really need to start to use the run tool in CoreRT? What are we getting out of it?

The run tool is deficient and confusing. I believe there are discussion about getting rid of it.